app/src/main/java/com/iti/android/team1/ebuy/model/datasource/repository/Repository.kt

```diff
@@ -93,10 +92,13 @@ class Repository(
         localSource.removeAllFavoriteProducts()
     }
 
-    override suspend fun addProductToFavorite(product: Product): DatabaseResponse<Long> {
+    override suspend fun addProductToFavorite(
+        product: Product,
+        noOfItems: Int,
```

Comment:
set default value in the repository for noOfItems

Reply:
An overriding function is not allowed to specify default values for its parameters, which means it is not possible to give a default value in the repository